works/models.py

```diff
@@ -0,0 +1,197 @@
+    current_status = models.ForeignKey(Status, related_name='works', on_delete=models.CASCADE)
+    work_type = models.ForeignKey(WorkType, related_name='works', on_delete=models.CASCADE)
+    iguala = models.ForeignKey(Iguala, related_name='works', on_delete=models.CASCADE,
+                               blank=True, null=True)
+
+    creation_date = models.DateField()
+    name = models.CharField(max_length=100)
+    expected_delivery_date = models.DateField()
+    brief = models.TextField()
+    final_link = models.CharField(max_length=1000, blank=True)
+
+
+class ArtWork(models.Model):
+    """ Model that represents the relation between a Work and its kind and quantity of arts.
+
+    Attributes:
+    -----------
+    work: ForeignKey
+        Relation with Work model.
+    art_type: ForeignKey
+        Relation with the art type requested by the work.
+    quantity: IntegerField
+        The quantity of that specific art type in that work.
+    """
+    work = models.ForeignKey(Work, related_name='art_works', on_delete=models.CASCADE)
+    art_type = models.ForeignKey(ArtType, related_name='art_works', on_delete=models.CASCADE)
+
+    quantity = models.IntegerField()
+
+
+class File(models.Model):
+    """ Model that represents a file uploaded by a user in a work.
+
+    Attributes:
+    -----------
+    work: ForeignKey
+        Relation with the work that the file belongs to.
+    upload: FileField
+        File to upload.
+    """
+    work = models.ForeignKey(Work, related_name='files', on_delete=models.CASCADE)
+
+    upload = models.FileField(upload_to='work_files/')
+
+
+class WorkDesigner(models.Model):
+    """ Model that represents the assignation of a designer to a work.
+
+    Attributes:
+    -----------
+    designer: ForeignKey
+        Relation with User model with the designer that will be assigned to the work.
+    work: ForeignKey
+        Work in which the designer will be assigned.
+    start_date: DateField
+        The date when the designer was assigned to the work.
+    end_date: DateField
+        The date when the designer was unassigned to the work.
+    """
+    designer = models.ForeignKey(User, related_name='asigned_works', on_delete=models.CASCADE)
+    work = models.ForeignKey(Work, related_name='work_designers', on_delete=models.CASCADE)
+
+    start_date = models.DateTimeField()
+    end_date = models.DateTimeField()
+
+
+class StatusChange(models.Model):
+    """ Model that represents the historical change of status of a work.
+
+    Attributes:
+    -----------
+    work: ForeignKey
+        Relation with the work.
+    status: ForeignKey
+        Relation with the Status model that indicates the status in that date.
+    """
+    work = models.ForeignKey(Work, related_name='status_changes', on_delete=models.CASCADE)
+    status = models.ForeignKey(Status, related_name='status_changes', on_delete=models.CASCADE)
+
+    date = models.DateTimeField()
```

Comment:
I know that it is not part of this code review, but I think the next task should be working in all the logic needed to change the status of a project. The signals.